lib/item_repository.rb

```diff
@@ -1,12 +1,19 @@
 require_relative 'item'
 require 'csv'
 require 'bigdecimal'
+require 'time'
 
 class ItemRepository
   def initialize(filepath)
     @items = build_items(filepath)
   end
 
+  # :nocov:
```

Comment:
@jbranham1 what is this?

Reply:
@abreaux26 it ignores inspect when you run simplecov!

Reply:
We may want to ask about this, but for now this is what google told me.